src/components/common/format.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react'
+import { DateTime } from 'luxon'
+
+const FormatDate = ({ date, format = 'ccc LLLL d yyyy' }) => {
+  if (typeof date === 'undefined') {
+    return null
+  }
+  return <>{DateTime.fromISO(date).toFormat(format)}</>
+}
+
+const FormatNumber = ({ number }) => (
+  <>
+    {number ? number.toString().replace(/\B(?=(\d{3})+(?!\d))/g, ',') : 'N/A'}
```

Comment:
I think we were not using t`oLocaleString()` because it throws errors when number is null. I'm going to add more logic to handle all edge cases then we can use `toLocaleString()`